HLTrigger/Configuration/python/customizeHLTforPatatrack.py

```diff
@@ -540,46 +650,69 @@ def customiseEcalLocalReconstruction(process):
         process.hltEcalRecHit = SwitchProducerCUDA(
             cpu = process.hltEcalRecHit.clone(
                 triggerPrimitiveDigiCollection = ('hltEcalDigisLegacy', 'EcalTriggerPrimitives')
-            ),
-            cuda = process.hltEcalRecHit.clone(
-                triggerPrimitiveDigiCollection = 'unused'
             )
         )
 
+    elif not hasattr(process.hltEcalRecHit, 'cpu'):
+        raise Exception('unsupported configuration: "process.hltEcalRecHit" is a SwitchProducerCUDA, but does not have a "cpu" branch')
+
+    if not hasattr(process.hltEcalRecHit, 'cuda'):
+        process.hltEcalRecHit.cuda = process.hltEcalRecHit.cpu.clone(
+            triggerPrimitiveDigiCollection = 'unused'
+        )
+
```

Comment:
> a bugfix for the handling of `process.hltEcalRecHit.cuda` (the bug was introduced in #36536).

After #36536, `process.hltEcalRecHit.cuda` would be added only if `process.hltEcalRecHit` was not a SwitchProducer in the input configuration. This condition is false in the latest menus, so for those `process.hltEcalRecHit.cuda` was not being added.